src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -17,7 +17,16 @@
           <dt>Orchestrator</dt>
           <dd i18n>{{ detailsCardData.orchestrator || 'Orchestrator is not available' }}</dd>
           <dt>Ceph version</dt>
-          <dd>{{ detailsCardData.cephVersion }}</dd>
+          <dd [ngClass]="{'mb-0': (upgradeInfo$ | async)}">{{ detailsCardData.cephVersion }}</dd>
+          <dd *ngIf="upgradeInfo$ | async as info"
+              i18n>
+            <a *ngIf="info.versions.length > 0"
+               (click)="upgradeModal()"
+               href="#/upgrade">
+              <i [ngClass]="icons.up"></i>
+              Upgrade available
+            </a>
```

Comment:
can you help, what happens when the cluster is upgrading? Do we still say Upgrade available?

can we change it to progress icon and saying upgrading?

Reply:
I'll look into it